odoo_sh/advanced/storage.rst

```diff
@@ -0,0 +1,50 @@
+:banner: banners/odoo-sh.jpg
+
+==================
+Storage Management
+==================
+
+Several elements take space in odoo.sh instances. This include:
+ - The production database
+ - The backups databases
+ - The staging builds
+
+More details for your instance can be found in the :ref:`Settings <odoosh-gettingstarted-settings-storage-size>`.
+
+Free up memory
+==============
+
+Filestore
+---------
+The filestore is a folder which contains every media file of a database instance. It does include the following:
+ - Pictures (products thumbnails, users avatar, ...)
+ - Attached chatter files (invoice and quotation PDF, ...)
+ - Minified CSS/JS files needed for the website to render quicker
```

Comment:
Technical assets bundles: Minified CSS/JS files needed for the website to render quicker